### PR TITLE
Make some small changes

### LINKED ISSUE #222: Update Axis Names
## Description
Adapt the dashboard script to capitalise axis names and remove any underscores.

## Required Files
- Home.py

## User Story
As a user I need to see a clean and polished dashboard so I can trust the data.

### LINKED ISSUE #224: Adjust Daily Releases Graph
## Description
x axis should read 'Date', y axis should read 'Number Of Daily Releases'. 
Also dates should be in chronological order.

## Required Files
- Home.py

## User Story
As a user I need to see a clean and polished dashboard so I can trust the data.

### PR DESCRIPTION
**Pull request by annalisev**
Neaten up a few things in the Dashboard with axis Names.

Closes #224 
Closes #222

### CODE REVIEW

File: dashboard/Home.py [hunk 1]
```diff
@@ -272,16 +272,19 @@ def make_tag_chart(data_df: pd.DataFrame, sorting=True) -> alt.Chart:
     new_rating_df = filter_dates(rating_df, filtered_days, "release_date")
     new_tag_df = filter_tags(tag_df, filtered_tags, "tag_name")
     top_twenty_games = top_twenty_games.drop('rating', axis=1)
+
+    # print(top_twenty_games.keys)
```
Comment (hhm970) [inline dashboard/Home.py:276]: Make sure to remove commented-out code

**Review by hhm970 — COMMENTED** [review]
The functionality of the code looks sound and well-defined. Other than the commented code, I am satisfied with this!

File: dashboard/Home.py (hunk 1 above)
Comment (annalisev) [reply dashboard/Home.py:282]: Good spot Howard I will fix this now!

File: dashboard/Home.py [hunk 2]
```diff
@@ -278,16 +278,18 @@ def make_tag_chart(data_df: pd.DataFrame, sorting=True) -> alt.Chart:
     new_rating_df = filter_dates(rating_df, filtered_days, "release_date")
     new_tag_df = filter_tags(tag_df, filtered_tags, "tag_name")
     top_twenty_games = top_twenty_games.drop('rating', axis=1)
+
+    # top_twenty_games['Ranking'] = top_twenty_games[0]+1
```
Comment (annalisev) [inline dashboard/Home.py:282]:
```suggestion
```

File: dashboard/pages/Epic.py [hunk 3]
```diff
@@ -48,7 +48,7 @@
     st.title("Epic Games Summary")
     st.write("---")
     st.subheader(
-        "_The latest metrics & graphs!_")
```
Comment (hhm970) [inline dashboard/pages/Epic.py:51]: Good that the underscores are being removed, to improve eligbility

File: dashboard/Home.py [hunk 4]
```diff
@@ -278,16 +278,17 @@ def make_tag_chart(data_df: pd.DataFrame, sorting=True) -> alt.Chart:
...
-    c_chart = count_chart(new_count_df)
+    c_chart = count_chart(new_count_df, sorting=False)
     r_chart = rating_chart(new_rating_df)
 
     st.subheader("This Weeks Most Popular Gaming Tags")
     st.altair_chart(tag_chart, use_container_width=True)
     st.subheader("Average Price Per Day",)
     st.altair_chart(p_chart, use_container_width=True)
-    st.subheader("This Weeks Top Twenty Games")
+    st.subheader("This Weeks Top 20 Games")
```
Comment (hhm970) [inline dashboard/Home.py:291]:
```suggestion
    st.subheader("This Week's Top 20 Games")
```

File: dashboard/pages/GOG.py [hunk 5]
```diff
@@ -93,7 +93,7 @@
 
     tag_chart = make_tag_chart(new_tag_df)
     p_chart = price_chart(new_price_df)
-    c_chart = count_chart(new_count_df)
+    c_chart = count_chart(new_count_df, sorted_=False)
     r_chart = rating_chart(new_rating_df)
 
     st.subheader("This Weeks Top Ten Games")
```
Comment (hhm970) [inline dashboard/pages/GOG.py:99]:
```suggestion
    st.subheader("This Week's Top Ten Games")
```

File: dashboard/pages/Steam.py [hunk 6]
```diff
@@ -90,7 +90,7 @@
 
     tag_chart = make_tag_chart(new_tag_df)
     p_chart = price_chart(new_price_df)
-    c_chart = count_chart(new_count_df)
+    c_chart = count_chart(new_count_df, sorted_=False)
     r_chart = rating_chart(new_rating_df)
 
     st.subheader("This Weeks Top Ten Games")
```
Comment (hhm970) [inline dashboard/pages/Steam.py:96]:
```suggestion
    st.subheader("This Week's Top Ten Games")
```

**Review by hhm970 — APPROVED** [review]
The code looks well-documented and coherent to me - well done!

**Pull request merged**